sycl/include/CL/sycl/detail/pi.h

```diff
@@ -315,7 +315,8 @@ typedef enum {
   PI_EXT_ONEAPI_DEVICE_INFO_MAX_GLOBAL_WORK_GROUPS = 0x20000,
   PI_EXT_ONEAPI_DEVICE_INFO_MAX_WORK_GROUPS_1D = 0x20001,
   PI_EXT_ONEAPI_DEVICE_INFO_MAX_WORK_GROUPS_2D = 0x20002,
-  PI_EXT_ONEAPI_DEVICE_INFO_MAX_WORK_GROUPS_3D = 0x20003
+  PI_EXT_ONEAPI_DEVICE_INFO_MAX_WORK_GROUPS_3D = 0x20003,
+  PI_EXT_ONEAPI_DEVICE_INFO_CUDA_ASYNC_BARRIER = 0x30000,
```

Comment:
I'd like somebody from the implementation side to take a look at this. This seems a big enough gap to not cause issues later, but I don't know if there's an established convention for experimental extensions. @smaslov-intel?

Reply:
What is this gap is intended to address? I'd prefer we don't have gaps and return errors from plugins that do not support any specific query.

Reply:
It makes it possible to keep consecutive numbers for similar aspects. In this case it allows adding more aspects from general extensions without mixing them with this, CUDA specific one. I can also remove the gap if that is preferred.

Reply:
Yes, please remove the gaps.